packages/react-components/src/components/MessageThread.tsx

```diff
@@ -429,7 +429,7 @@ export type MessageThreadProps = {
    * Optional callback to override render of a message.
    *
    * @param messageProps - props of type MessageProps
-   * @param defaultOnRender - default render of type DefaultMessageRendererType
+   * @param defaultOnRender - default render of type MessageRenderer
```

Comment:
nit: we should use `@link` more for referencing other parts of packages - then we can have api-extractor break when links to do not resolve correctly https://api-extractor.com/pages/messages/ae-unresolved-link/ so renames are caught